### PR TITLE
#159428360 Connect api endpoint to view details of an entry to front end 

### PR DESCRIPTION
**Pull request by kampkelly**
#### What does this PR do?
This pull request connects the api endpoint to view details of an entry to the front-end page to show an entry.

#### Description of Task to be completed?
The following tasks have been completed:
A user can click on an entry to see more details of that entry
A user can delete an entry

#### How should this be manually tested?
After cloning the repo, run npm install, then open show.html from the Client folder in any browser of your choice, preferably chrome.

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#159428360:(https://www.pivotaltracker.com/story/show/159428360)

### CODE REVIEW

File: Client/javascripts/es6/script.js [hunk 1]
```diff
@@ -173,6 +173,36 @@ function forgotPassword() {
 	}
 }
 
+// eslint-disable-next-line
```
Comment (phemmz) [inline Client/javascripts/es6/script.js:176]: Why are you disabling eslint here?

File: Client/javascripts/es6/show.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+					fetch(`${baseUrl}/entries/${entryId}`, {
+						method: 'DELETE',
+						headers: {
+							'Content-Type': 'application/json',
+							token: localStorage.getItem('diary_token'),
+						},
+					})
+						.then(res => res.json())
+						.then((newData) => {
+							console.log(newData);
```
Comment (phemmz) [inline Client/javascripts/es6/show.js:58]: console.log???

File: Client/javascripts/es6/show.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+			token: localStorage.getItem('diary_token'),
+		},
+	})
+		.then(res => res.json())
+		.then((data) => {
+			console.log(data);
+			document.querySelector('body').insertAdjacentHTML('afterbegin', '<img src="images/Rolling.svg" id="loading" />');
+			if (data.status === 'Failed') {
+				document.getElementById('loading').style.display = 'none';
+				document.getElementById('flash-message').style.display = 'block';
```
Comment (phemmz) [inline Client/javascripts/es6/show.js:19]: You can define your selectors as a global variable and then reuse them wherever you want to

File: Client/javascripts/es6/script.js (hunk 1 above)
Comment (kampkelly) [reply Client/javascripts/es6/script.js:176]: Eslint was saying 'viewEntries' variable has been defined but is not being used. But actually it is called from an onclick event on the browser.

File: Client/javascripts/es6/show.js (hunk 2 above)
Comment (kampkelly) [reply Client/javascripts/es6/show.js:58]: It will be removed.

File: Client/javascripts/es6/show.js (hunk 3 above)
Comment (kampkelly) [reply Client/javascripts/es6/show.js:19]: Okay, will do so

**Pull request merged**